recipes/ogdf/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "ogdf" %}
+{% set version = "2020.02" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ogdf/ogdf/archive/catalpa-202002.tar.gz
+  sha256: 32f3f943675e55d1a23e62293d016dc1644b1176271d60f41788ade60c2d3fc5
+
+build:
+  number: 0
+  skip: True  # [win]
```

Comment:
This should have a run_exports

Reply:
I read the doc about it. Not sure I understand everything so let me know if what I did is correct.

Reply:
This will work because it only has major & minor numbers, but we usually do something like : `{{ pin_subpackage('ogdf', max_pin='x.x') }}`

Reply:
Thanks for the explanation.

Reply:
This still needs a run_exports

Reply:
> This still needs a run_exports

if OGDF is built as a shared lib.

Reply:
I will build ogdf as a shared lib.